mne/connectivity/spectral.py

```diff
@@ -699,60 +699,47 @@ def spectral_connectivity(data, method='coh', indices=None, sfreq=2 * np.pi,
             C = ---------------------
                 sqrt(E[Sxx] * E[Syy])
 
-        'imcoh' : Imaginary coherence [1]_ given by::
+        'imcoh' : Imaginary coherence :footcite:`NolteEtAl2004`
+        given by::
 
                       Im(E[Sxy])
             C = ----------------------
                 sqrt(E[Sxx] * E[Syy])
-
-        'plv' : Phase-Locking Value (PLV) [2]_ given by::
-
+        'plv' : Phase-Locking Value (PLV)
+        :footcite:`LachauxEtAl1999` given by::
```

Comment:
you removed a blank line above here that needs to be there to distinguish the indented block (the formula) with the text that comes below

```suggestion

        'plv' : Phase-Locking Value (PLV)
        :footcite:`LachauxEtAl1999` given by::
```

Reply:
@apoorva6262 you are still missing this blank line, I think. That's why CIs did not come back green. (You added one in line 710 but not 708)